qir/qsharp/TargetPackages/QSharpVersion/Example.csproj

```diff
@@ -1,19 +1,13 @@
-<Project Sdk="Microsoft.Quantum.Sdk/0.24.41925-alpha">
+<Project Sdk="Microsoft.Quantum.Sdk/0.24.41975-alpha">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
-    <QirGeneration>true</QirGeneration>
-    <AdditionalQscArguments>--runtime-override AdaptiveExecution</AdditionalQscArguments>
-    <IncludeCSharpRuntime>False</IncludeCSharpRuntime>
-    <IncludeQSharpCorePackages>False</IncludeQSharpCorePackages>
     <IncludeProviderPackages>False</IncludeProviderPackages>
-    <CSharpGeneration>False</CSharpGeneration>
-
+    <ExecutionTarget>qci.qpu</ExecutionTarget>
+    <!--ExecutionTarget>rigetti.qpu</ExecutionTarget-->
+    <!--TargetCapability>AdaptiveExecution</TargetCapability-->
+    <QirGeneration>true</QirGeneration>    
```

Comment:
Same here. This is not needed.

Reply:
I use this project for developing tests, so it is kind of handy to have here.